core/amber/src/main/protobuf/edu/uci/ics/amber/engine/common/ambermessage.proto

```diff
@@ -21,11 +21,11 @@ message ControlPayloadV2 {
 }
 
 message PythonDataHeader {
-  core.ActorVirtualIdentity tag = 1 [(scalapb.field).no_box = true];
```

Comment:
I don't see the need here.

ChannelID = (sender actor id + receiver actor id). receiver actor id is always this actor itself. The only thing changing is the sender actor id. So I think this change does not bring any additional information?

Reply:
As discussed, we need to align both ECM and ECM, which are from different channels of the same actor.